docker-compose.yml

```diff
@@ -984,14 +988,17 @@ services:
     # Parameters:
     #   ARCH: amd64, arm64v8, ...
     #   UBUNTU: 20.04, 22.04
-    image: ${REPO}:${ARCH}-ubuntu-${UBUNTU}-python-3
+    #   PYTHON: 3.9, 3.10, ...
+    image: ${REPO}:${ARCH}-ubuntu-${UBUNTU}-python-${PYTHON}
```

Comment:
Again, I don't think these changes are desirable. The purpose of `ubuntu-python` is to check against the Ubuntu-provided Python, not a self-compiled Python.

Reply:
(same as `fedora-python`, `debian-python`, actually)